python/ray/_raylet.pyx

```diff
@@ -158,6 +158,16 @@ OPTIMIZED = __OPTIMIZE__
 
 logger = logging.getLogger(__name__)
 
+# A helper function to get whether the actor has any async method.
+def actor_has_async_methods(actor_class):
```

Comment:
Forgot to delete `actor_has_async_methods` in `actor.py`?

Reply:
The following option `max_concurrency` depends on it.

But it's ok to remove, and then we need move setting default `max_concurrency` logic to callee side as well.